tests/unit/models/360-core.js

```diff
@@ -158,6 +158,33 @@ describe("SerSol 360 Core Model >", function() {
     });
   });
 
+  describe("serSol360Core model libraryIdOverride method staging >", function() {
+    beforeEach(function() {
+      delete browzine.api;
+    });
+
+    afterEach(function() {
+      delete browzine.libraryId;
+      browzine.api = "https://staging-api.thirdiron.com/public/v1/libraries/XXX";
+    });
+
+    it("should not override the libraryId on the api endpoint when the libraryId is not specified", function() {
+      browzine.api = "https://staging-api.thirdiron.com/public/v1/libraries/XXX";
+      expect(serSol360Core.libraryIdOverride(serSol360Core.urlRewrite(browzine.api))).toEqual("https://staging-api.thirdiron.com/public/v1/libraries/XXX");
+    });
+
+    it("should override the libraryId on the api endpoint even when an api endpoint is specified", function() {
+      browzine.api = "https://staging-api.thirdiron.com/public/v1/libraries/XXX";
+      expect(serSol360Core.libraryIdOverride(serSol360Core.urlRewrite(browzine.api))).toEqual("https://staging-api.thirdiron.com/public/v1/libraries/XXX");
+    });
```

Comment:
Do we need to change this test?  The `it` seems to be a scenario where `libraryId` is specified in addition to `api` - is that right?

The `afterEach` call deletes `browzine.libraryId` from the object, and there's nothing in the `it` or the `describe` that sets `libraryId`

So do we want it set here, perhaps like this?

```suggestion
    it("should override the libraryId on the api endpoint even when an api endpoint is specified", function() {
      browzine.api = "https://staging-api.thirdiron.com/public/v1/libraries/XXX";
      browzine.libraryId = '123Y';
      expect(serSol360Core.libraryIdOverride(serSol360Core.urlRewrite(browzine.api))).toEqual("https://staging-api.thirdiron.com/public/v1/libraries/XXX");
    });
```